### PR TITLE
feat: initial version of fail2ban template

### PR DESCRIPTION
**Pull request by rhajek**
This initial version of fail2ban template.

### CODE REVIEW

**Review by xe-nvdk — CHANGES_REQUESTED** [review]
Looks good to me. Please, can you modify the main readme and add info about your template?

File: fail2ban/readme.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+#### InfluxDB UI
+
+In the InfluxDB UI, go to Settings->Templates and enter this URL: https://raw.githubusercontent.com/influxdata/community-templates/master/fail2ban/img/fail2ban-template.yml
+
+#### Influx CLI
+If you have your InfluxDB credentials configured in the CLI, you can install this template with:
+
+`influx apply -f https://raw.githubusercontent.com/influxdata/community-templates/master/fail2ban/fail2ban-template.yml`
+
+## Requirements
```
Comment (mhall119) [inline fail2ban/readme.md:18]: Can you move this section into the `## Setup Instructions` section? That way they will be shown in the UI when somebody is installing this template.

**Review by mhall119 — CHANGES_REQUESTED** [review]
One small change to the readme.md to give a good user experience when installing via the GUI

File: fail2ban/readme.md (hunk 1 above)
Comment (rhajek) [reply fail2ban/readme.md]: Readme.md was updated.

**Review by xe-nvdk — APPROVED** [review]

**Review by mhall119 — APPROVED** [review]
Approving with Ignacio's review

**Pull request merged**